### PR TITLE
Remove hex encoding in manager-backup-cassandra

### PR DESCRIPTION
**Pull request by rinx**
### Description:

I've just removed the hex encoding process in manager-backup-casssandra.

### Related Issue:

Nothing.

### How Has This Been Tested?:

In my local kind cluster.

### Environment:

- Golang Version: 1.13.5
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.8.4

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [X] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kpango** [comment]
gozstd Reader & Writer has stream method such as WriteTo & ReadFrom please use stream method which is more efficient for performance

**Comment by rinx** [comment]
@kpango I've just fixed it.

File: internal/compress/zstd.go [hunk 1]
```diff
@@ -44,21 +42,36 @@ func NewZstd(opts ...ZstdOption) (Compressor, error) {
 }
 
 func (z *zstdCompressor) CompressVector(vector []float32) ([]byte, error) {
+	buf := new(bytes.Buffer)
+	zw := gozstd.NewWriterLevel(buf, z.compressionLevel)
+
 	gob, err := z.gobc.CompressVector(vector)
 	if err != nil {
 		return nil, err
 	}
```
Comment (kpango) [inline internal/compress/zstd.go]: could you please initialize gozstd here

Comment (rinx) [reply internal/compress/zstd.go]: i've just reordered it.

**Pull request merged**